src/Learning/KWLearningProblem/KWLearningProblemComputeStats.cpp

```diff
@@ -116,9 +116,7 @@ void KWLearningProblem::ComputeStats()
 	KWLearningErrorManager::AddTask("Data preparation");
 
 	// Initialisation des objets de calculs des statistiques
```

Comment:
Perhaps update the comment to something like:
```c++
// Initialisation des objets de calculs des statistiques dans tous les cas (meme en cas d'erreur)
```

Reply:
Fait